stretch_core/stretch_core/action_exceptions.py

```diff
@@ -0,0 +1,15 @@
+from control_msgs.action import FollowJointTrajectory
+
+
+# Exceptions for dealing with FollowJointTrajectory
+class FollowJointTrajectoryException(RuntimeError):
+    pass
```

Comment:
Is this a placeholder?

Reply:
If not I think you can assing `CODE` to your arbitrary code (-100)  from the earlier PR